{{cookiecutter.project_slug}}/android/fastlane/Fastfile

```diff
@@ -12,36 +12,40 @@
 # Opt out of sending fastlane usage metrics
 opt_out_usage
 
-fastlane_require 'httparty'
-
-base_name = "{{cookiecutter.project_slug}}"
+fastlane_require('httparty')
 
 default_platform(:android)
 
-platform :android do
-  desc "Runs all the tests"
-  lane :test do
-    gradle(task: "test")
+base_name = '{{cookiecutter.project_slug}}'
+
+platform(:android) do
+
```

Comment:
Extra newline here isn't necessary

Reply:
sorry, that's probably added there by mistake